payroll/views.py

```diff
@@ -174,6 +177,50 @@ def get_context_data(self, **kwargs):
         return context
 
 
+class DownloadView(TemplateView):
+    def get(self, request, *args, **kwargs):
+        slug = request.GET.get('employer')
+        year = request.GET.get('year')
+        employer = Employer.objects.get(slug=slug)
+        employer_salaries = employer.get_salaries(year=year)
+
+        rows = []
+
+        for salary in employer_salaries:
+            name_kwargs = {
+                'first_name': salary.job.person.first_name,
+                'last_name': salary.job.person.last_name
+            }
+            name = '{first_name} {last_name}'.format(**name_kwargs)
+
+            start_date = salary.job.start_date.strftime('%m/%d/%Y') if salary.job.start_date else ''  # noqa
+
+            row = OrderedDict([
```

Comment:
what determines the column order is the order of the fields in `fieldnames` argument, and that's it. 

you are already explicitly setting the order you want every time you are creating the ordered dict. it's simpler and clearer to just do it once when you define `fieldnames`

Reply:
> what determines the column order is the order of the fields in `fieldnames` argument, and that's it.

@fgregg yeah, i understand that. that's why i changed it to an ordered dict, so i can do this: `fieldnames=rows[0].keys()` instead of writing out an array there. i personally prefer that way because i'm only writing the key names once (in the ordered dict), but i'll change it back to the way you're suggesting.

Reply:
i see. I understand your point about duplication, but this approach is more more common in python and will allow a streaming approach that may be very useful for performance later.

Reply:
makes sense. thanks for that clarification.